sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,736 @@
+from cryptography.hazmat.primitives import serialization
+from cryptography.x509 import load_pem_x509_certificate
 
-    from azure.core.credentials import TokenCredential
-    from azure.core.pipeline.transport import HttpRequest, HttpResponse
+from azure.core.tracing.decorator import distributed_trace
 
 from ._generated import AzureAttestationRestClient
 from ._generated.models import (
-    AttestationType, 
-    PolicyResult as GeneratedPolicyResult, 
-    PolicyCertificatesResult, 
-    JSONWebKey, 
-    AttestationCertificateManagementBody, 
+    AttestationType,
+    PolicyResult as GeneratedPolicyResult,
+    PolicyCertificatesResult as GeneratedPolicyCertificatesResult,
+    JSONWebKey,
+    AttestationCertificateManagementBody,
     StoredAttestationPolicy as GeneratedStoredAttestationPolicy,
-    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult
+    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult,
 )
 from ._configuration import AttestationClientConfiguration
 from ._models import (
-    AttestationSigner, 
-    AttestationToken, 
-    AttestationResponse, 
-    AttestationSigningKey, 
-    PolicyCertificatesModificationResult,
-    PolicyResult,
-    AttestationTokenValidationException
+    AttestationSigner,
+    AttestationToken,
+    AttestationPolicyResult,
+    AttestationPolicyCertificateResult,
 )
-import base64
-from azure.core.tracing.decorator import distributed_trace
-from threading import Lock, Thread
+from ._common import pem_from_base64, validate_signing_keys, merge_validation_args
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from azure.core.credentials import TokenCredential
 
 
 class AttestationAdministrationClient(object):
+    # pylint: disable = line-too-long
     """Provides administrative APIs for managing an instance of the Attestation Service.
 
-    :param instance_url: base url of the service
-    :type instance_url: str
+    The :class:`~azure.security.attestation.AttestationAdministrationClient` object implements the policy
+    management and policy certificate management functions.
+
     :param credential: Credentials for the caller used to interact with the service.
     :type credential: :class:`~azure.core.credentials.TokenCredential`
-
-    :keyword Pipeline pipeline: If omitted, the standard pipeline is used.
-    :keyword HttpTransport transport: If omitted, the standard pipeline is used.
-    :keyword list[HTTPPolicy] policies: If omitted, the standard pipeline is used.
+    :param str endpoint: The attestation instance base URI, for example https://mytenant.attest.azure.net.
+    :keyword str signing_key: PEM encoded signing key to be used for all
+        operations.
+    :keyword str signing_certificate: PEM encoded X.509 certificate to be used for all
+        operations.
+
+    :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
```

Comment:
nit: unnecessary line
```suggestion
        operations.
    :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
```